pysingfel/detector/base.py

```diff
@@ -301,16 +301,14 @@ def get_fxs_photons_slices(self, particles, beam_focus_radius, jet_radius, mesh_
         mesh, voxel_length= self.get_reciprocal_mesh(voxel_number_1d = mesh_length)
         state, coords = distribute_particles(particles, beam_focus_radius, jet_radius)
         count = 0
+        field_acc = np.zeros((4, 512, 512), dtype=np.complex128)
```

Comment:
do not hardcode pixel dimensions for pnccd. this function needs to work for all detector dimensions